Dockerfile-debian.template

```
@@ -147,5 +148,6 @@ RUN set -ex; \
 COPY *.sh upgrade.exclude /
 COPY config/* /usr/src/nextcloud/config/
 
+HEALTHCHECK CMD "/healthcheck.sh"
```

This seems overly complicated :)

```suggestion
HEALTHCHECK ["CMD --start-period=30s", "CMD", "curl", "--fail", "http://localhost/status.php"]
```